src/librustdoc/html/layout.rs

```diff
@@ -48,6 +48,7 @@ r##"<!DOCTYPE html>
     <title>{title}</title>
 
     <link rel="stylesheet" type="text/css" href="{root_path}main.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
There's no need to include the whole font-awesome package for one small rocket icon. This won't work for offline documentation, too.